packages/yoastseo/src/scoring/assessments/readability/SubheadingDistributionTooLongAssessment.js

```diff
@@ -222,161 +233,180 @@ class SubheadingsDistributionTooLong extends Assessment {
 	 * @returns {Array} The array containing subheading texts that are too long.
 	 */
 	getTooLongSubheadingTexts() {
-		return filter( this._subheadingTextsLength, function( subheading ) {
-			return subheading.countLength > this._config.parameters.recommendedMaximumLength;
-		}.bind( this ) );
+		return this._subheadingTextsLength.filter( subheading => subheading.countLength > this._config.parameters.recommendedMaximumLength );
```

Comment:
Similar to the above comment, [native `filter` is also faster compared to `lodash`](https://techspire.nl/applicatie-ontwikkeling/why-you-shouldnt-use-lodash/#:~:text=Performance&text=We%20can%20see%20that%20Lodash,than%20using%20the%20native%20code.). Although the main reason for this is so that we're using less lodash dependency in the package